components/raftstore/src/coprocessor/config.rs

```diff
@@ -98,16 +98,16 @@ impl Default for Config {
 impl Config {
     pub fn validate(&mut self) -> Result<()> {
         if self.region_max_size.0 < self.region_split_size.0 {
```

Comment:
How about making `region_max_size` as `Option<ReadableSize>`? So if it's not set, we can give it a correct number.

Reply:
Changed, PTAL again.